R/tt_pos_and_access.R

```diff
@@ -280,6 +280,34 @@ setMethod("tt_at_path", "VTableTree",
 
 #' @export
 #' @rdname ttap
+#' 
+#' @examples 
+#' # Accessing sub table.
+#' lyt <- basic_table() %>%
+#'    split_cols_by("ARM") %>%
+#'    split_rows_by("SEX") %>%
+#'    split_rows_by("BMRKR2") %>%
+#'    analyze("AGE")
+#'
+#' tbl <- build_table(lyt, ex_adsl)
+#' sub_tbl <- tt_at_path(tbl, path = c("SEX", "F", "BMRKR2"))
+#'
+#' # Removing sub table.
+#' tbl2 <- tbl
+#' tt_at_path(tbl2, path = c("SEX", "F")) <- NULL
+#' tbl2
+#'
+#' # Setting sub table.
+#' lyt3 <- basic_table() %>%
+#'    split_cols_by("ARM") %>%
+#'    split_rows_by("SEX") %>%
+#'    analyze("BMRKR2")
+#'
+#' tbl3 <- build_table(lyt3, ex_adsl)
```

Comment:
```suggestion
#' tbl3 <- build_table(lyt3, ex_adsl) %>% prune_table()
```
Just better for clarity